react/README.md

```diff
@@ -0,0 +1,158 @@
+# Introduction
+
+This React sample demonstrates how to perform login actions that will initiate flow endpoint operations are used only to implement custom authentication UIs. 
+OIDC/OAuth 2 requests initiate the flow and redirect the browser to the custom authentication UI (which is configured in the application through the application’s `loginPageUrl` property.)
+
+This example contains code for SPA that( by default ) calls OAuth 2.0 implicit flow, where the access token is returned immediately without an extra authorization code exchange step.
+If succeeded, the  OAuth 2.0 protected resource ( [UserInfo Endpoint](https://apidocs.pingidentity.com/pingone/customer/v1/api/auth/p1-a_Authorize/#UserInfo-endpoint) ) that returns claims about the authenticated end user can be tested by clicking __"Show user information"__ link.
+
+You can also try other authorization grants like `authorization_code` or `client_credentials`, where your application will call the `/{environmentId}/as/token` endpoint to acquire the access token.
+For this grant types the `grantType` and `clientSecret` in [config.js](src/config.js) are necessary.
+
+# Content
+- [Prerequisites](#prerequisites)
+- [Getting Started](#getting-started)
+- [Developer Notes](#developer-notes)
+- [Available Scripts](#available-scripts)
+
+# Prerequisites 
+You will need the following things:
+
+- PingOne for Customers Account - If you don’t have an existing one, please register it.
+- An new [Application](https://documentation.pingidentity.com/pingone/p14cAdminGuide/index.shtml#p1_t_addApplication.html), configured for Single-Page App (SPA) or Native mode. Also make sure that it is enabled plus redirect, sign off URL's and access grants by scopes are properly set.
```

Comment:
A PingCloud application [..]. also make sure that it is enabled, the redirect and sign off urls are set, and resource access grants are properly set.

Reply:
We are using PingOne for now